express/features/sign-in-unhappy-paths.feature

```diff
@@ -3,6 +3,7 @@ Feature: Unhappy paths for the self-service sign-in flow
   Background:
     Given that the user is on the "/sign-in" page
 
```

Comment:
Undo spacing (we do want an empty line at the end of files)

Reply:
Done.